src/components/postBoard.tsx

```diff
@@ -5,14 +5,40 @@ import Post from './post';
 
 class PostBoard extends Component {
 
+    state = {
+        userPosts: [
+            {
+                userId: 0,
+                id: 0,
+                title: "None",
+                completed: false
+            }
+        ]
+    };
+
+    componentDidMount(){
+     
+        fetch('https://jsonplaceholder.typicode.com/posts')
+        .then(response => response.json())
+        .then(json => {
+            let userPosts = json;
+            this.setState({userPosts});
+        });
```

Comment:
Try using js await syntax here. It's becoming more popular: https://www.valentinog.com/blog/await-react/